openlibrary/tests/accounts/test_models.py

```diff
@@ -1,7 +1,28 @@
-from openlibrary.accounts import model
+from openlibrary.accounts import model, InternetArchiveAccount
```

Comment:
Why not `from openlibrary.accounts.InternetArchiveAccount import xauth` and then lose lines 13, 14, 22, 23?

Reply:
:+1: I think I was initially thinking to avoid polluting the module's namespace since there are a few classes in the file which could have tests, but it's fine. Long term this should probably be refactored so there aren't so many things in that file :P

Reply:
Actually my IDE won't let me do that; maybe because `InternArchiveAccount` is a class? `xauth` is a static method.